OverlayPlugin.Core/MemoryProcessors/Party/PartyMemory.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace RainbowMage.OverlayPlugin.MemoryProcessors.Party
+{
+    public class PartyListEntry
```

Comment:
I do think a lot of the position, hp/mp, content/territory, sex etc members can all be dropped here as they're either not useful / in other lines.

Reply:
There's no real cost to having it here, this is a transitory class to copy the data from the raw pointer mapped class. This info isn't included in the `OnPartyChange` event (except in the `#ifdef DEBUG` debug variable addition).